<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QMainWindow" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>分组框</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QWidget" name="widgetControl" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="1" rowspan="2">
           <widget class="QSlider" name="verticalSlider">
            <property name="value">
             <number>80</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="3" rowspan="2">
           <widget class="QScrollBar" name="verticalScrollBar">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="2">
           <widget class="QProgressBar" name="progressBar2">
            <property name="maximumSize">
             <size>
              <width>12</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="value">
             <number>80</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="4" column="4">
          <widget class="QDoubleSpinBox" name="doubleSpinBox">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="btnTip">
           <property name="text">
            <string>右下框</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="rbtn2">
           <property name="text">
            <string>单选框2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="btnStyle3">
           <property name="text">
            <string>软件黑</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3" colspan="3">
          <widget class="QLineEdit" name="txt1">
           <property name="text">
            <string>拿人钱财替人消灾,人生江湖如此,程序江湖亦如此.</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QComboBox" name="cbox1"/>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btnInfo">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>信息框</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLineEdit" name="txt2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>只读文本框</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="5">
          <widget class="QDateTimeEdit" name="dateTimeEdit">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLineEdit" name="txt3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>1234</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="btnDate">
           <property name="text">
            <string>日期框</string>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QDateEdit" name="dateEdit">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="5">
          <widget class="QFontComboBox" name="fontComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="btnQuestion">
           <property name="text">
            <string>询问框</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLCDNumber" name="lcdNumber">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="digitCount">
            <number>8</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
           <property name="value" stdset="0">
            <double>12345678.000000000000000</double>
           </property>
           <property name="intValue" stdset="0">
            <number>12345678</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btnStyle1">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>浅蓝色</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="btnError">
           <property name="text">
            <string>错误框</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QSpinBox" name="spinBox"/>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="lab">
           <property name="text">
            <string>我是标签</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QComboBox" name="cbox2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>我是标签</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="btnWidget">
           <property name="text">
            <string>新窗体</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="rbtn1">
           <property name="text">
            <string>单选框1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="btnStyle2">
           <property name="text">
            <string>扁平灰</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="btnSplash">
           <property name="text">
            <string>提示框</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="ck3">
           <property name="text">
            <string>复选框3</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="ck2">
           <property name="text">
            <string>复选框2</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="ck1">
           <property name="text">
            <string>复选框1</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>单选框3</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QPushButton" name="btnAboutQt">
           <property name="text">
            <string>关于Qt</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="btnAbout">
           <property name="text">
            <string>关于程序</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QSlider" name="horizontalSlider">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="4">
          <widget class="QProgressBar" name="progressBar1">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>12</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="5" column="5">
          <widget class="QScrollBar" name="horizontalScrollBar">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCalendarWidget" name="calendarWidget2">
        <property name="gridVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9.07563pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <item>
         <property name="text">
          <string>新建项目</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
        </item>
        <item>
         <property name="text">
          <string>新建项目</string>
         </property>
         <property name="checkState">
          <enum>PartiallyChecked</enum>
         </property>
        </item>
        <item>
         <property name="text">
          <string>新建项目</string>
         </property>
         <property name="checkState">
          <enum>Unchecked</enum>
         </property>
        </item>
        <item>
         <property name="text">
          <string>新建项目</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
        </item>
        <item>
         <property name="text">
          <string>新建项目</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <column>
         <property name="text">
          <string>新建列</string>
         </property>
        </column>
        <item>
         <property name="text">
          <string>新建项目</string>
         </property>
         <property name="checkState">
          <enum>PartiallyChecked</enum>
         </property>
         <item>
          <property name="text">
           <string>新建子项目</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建子项目</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>新建子项目</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>新建项目</string>
         </property>
         <property name="checkState">
          <enum>Checked</enum>
         </property>
         <item>
          <property name="text">
           <string>新建子项目</string>
          </property>
          <property name="checkState">
           <enum>Checked</enum>
          </property>
          <item>
           <property name="text">
            <string>新建子项目</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
          </item>
         </item>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>新建列</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>新建列</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>新建列</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>新建列</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>新建列</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>11111</string>
       </property>
       <property name="checkState">
        <enum>Checked</enum>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>11111</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>11111</string>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>11111</string>
       </property>
      </item>
      <item row="0" column="4">
       <property name="text">
        <string>11111</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>22222</string>
       </property>
       <property name="checkState">
        <enum>Checked</enum>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>22222</string>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>22222</string>
       </property>
      </item>
      <item row="1" column="3">
       <property name="text">
        <string>22222</string>
       </property>
      </item>
      <item row="1" column="4">
       <property name="text">
        <string>22222</string>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>33333</string>
       </property>
       <property name="checkState">
        <enum>Checked</enum>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>33333</string>
       </property>
      </item>
      <item row="2" column="2">
       <property name="text">
        <string>33333</string>
       </property>
      </item>
      <item row="2" column="3">
       <property name="text">
        <string>33333</string>
       </property>
      </item>
      <item row="2" column="4">
       <property name="text">
        <string>33333</string>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>44444</string>
       </property>
       <property name="checkState">
        <enum>Checked</enum>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string>44444</string>
       </property>
      </item>
      <item row="3" column="2">
       <property name="text">
        <string>44444</string>
       </property>
      </item>
      <item row="3" column="3">
       <property name="text">
        <string>44444</string>
       </property>
      </item>
      <item row="3" column="4">
       <property name="text">
        <string>44444</string>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string>55555</string>
       </property>
       <property name="checkState">
        <enum>Checked</enum>
       </property>
      </item>
      <item row="4" column="1">
       <property name="text">
        <string>55555</string>
       </property>
      </item>
      <item row="4" column="2">
       <property name="text">
        <string>55555</string>
       </property>
      </item>
      <item row="4" column="3">
       <property name="text">
        <string>55555</string>
       </property>
      </item>
      <item row="4" column="4">
       <property name="text">
        <string>55555</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_O"/>
    <addaction name="separator"/>
    <addaction name="action_S"/>
    <addaction name="action_C"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_X"/>
    <addaction name="action_C_2"/>
    <addaction name="action_V"/>
   </widget>
   <widget class="QMenu" name="menu_B">
    <property name="title">
     <string>构建(&amp;B)</string>
    </property>
    <widget class="QMenu" name="menu_C">
     <property name="title">
      <string>编译项目(&amp;C)</string>
     </property>
     <addaction name="action_1"/>
     <addaction name="action_2"/>
    </widget>
    <addaction name="actionGouj"/>
    <addaction name="menu_C"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_T"/>
    <addaction name="action_A"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_B"/>
   <addaction name="menu_H"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>新建文件(&amp;N)</string>
   </property>
  </action>
  <action name="action_O">
   <property name="text">
    <string>打开文件(&amp;O)</string>
   </property>
  </action>
  <action name="action_S">
   <property name="text">
    <string>保存文件(&amp;S) </string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>关闭退出(&amp;C)</string>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>剪切(&amp;X) </string>
   </property>
  </action>
  <action name="action_C_2">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="action_V">
   <property name="text">
    <string>粘贴(&amp;V)</string>
   </property>
  </action>
  <action name="actionGouj">
   <property name="text">
    <string>构建项目(&amp;B)</string>
   </property>
  </action>
  <action name="action_T">
   <property name="text">
    <string>帮助文档(&amp;T)</string>
   </property>
  </action>
  <action name="action_A">
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
  </action>
  <action name="action_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>编译项目1</string>
   </property>
  </action>
  <action name="action_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>编译项目2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBar1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBar2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>760</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
